Pocket/Pocket/Models/Transaction.swift

```diff
@@ -7,32 +7,14 @@
 //
 
 import Foundation
+import SwiftyJSON
 
-public struct Transaction {
-    public var nonce = ""
-    public var recipient = ""
-    public var value = ""
-    public var data = [AnyHashable: Any]()
-    public var signed = ""
+public class Transaction: PocketRequestProtocol {
+    public var network: Network?
+    public var serialized_tx = ""
```

Comment:
Do not use underscores - it's not in common in the Swift world. change this to: `serializedTransaction`.